contrib/babelfishpg_tsql/src/utils/removeaccent.map

```diff
@@ -1,6 +1,9 @@
 /* TRANSFORMATION_RULE "[[:Latin:][:Nd:]]; Latin-ASCII" */
 
-static const remove_accent_map_pair pltsql_remove_accent_map[811] = 
+#define pltsql_remove_accent_map_icu_major_version 60
+#define pltsql_remove_accent_map_icu_min_version 2
```

Comment:
Instead, let's create internal function which will help us getting version of ICU using which this map is created